src/Command.ts

```diff
@@ -27,6 +27,10 @@ const executors = {
     },
     clear: (invocation: Invocation, args: Array<string>): void => {
         setTimeout(() => invocation.terminal.clearInvocations(), 0);
+    },
+    exit: (invocation: Invocation, args: Array<string>): void => {
+        var app = require('remote').require('app');
+        app.quit();
```

Comment:
I guess we should make `Application` a singleton class accessible from everywhere. We'll have only one instance of it anyway.